update.js

```diff
@@ -1,30 +1,30 @@
 #!/usr/bin/env node
 
-'use strict'
-
-const fs = require('fs')
-const util = require('util')
-const path = require('path')
-const semver = require('semver')
-const ethUtil = require('ethereumjs-util')
-const ipfsImporter = require('ipfs-unixfs-importer')
-const IPLD = require('ipld')
-const inMemory = require('ipld-in-memory')
-const swarmhash = require('swarmhash')
-
-// This script updates the index files list.js and list.txt in the directories containing binaries,
-// as well as the 'latest' and 'nightly' symlinks/files.
-
-const ipfsHash = async (content) => {
-  const iterator = ipfsImporter.importer([{ content }], await inMemory(IPLD), { onlyHash: true })
-  const { value, done } = await iterator.next()
-  if (done) {
-    throw new Error('Failed to calculate an IPFS hash.')
-  }
-
-  await iterator.return()
-  return value.cid.toString()
-}
+//this is a workaround for the semver as far as it is a CommonJS module
+import semver from "semver"
+const { compare, gt } = semver
+
+import { fileURLToPath } from "url";
+import { dirname, join } from "path";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+import {
+  readlinkSync,
+  unlinkSync,
+  symlinkSync,
+  readFile,
+  readFileSync,
+  writeFile,
+  readdir,
+  stat,
+  lstat
+ } from "fs"
+import swarmhash from "swarmhash";
+import { promisify } from "util"
```

Comment:
```suggestion
import { readFile } from 'node:fs/promises';
```
And remove `readFile` from the import above. Also, we could rename it to `readFileAsync` to be explicit about the function implementation being used:

```javascript
import { readFile as readFileAsync } from 'node:fs/promises';
```